<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyFirstQt_VSClass</class>
 <widget class="QMainWindow" name="MyFirstQt_VSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyFirstQt_VS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="show_hello_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>切换hello world状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="hello_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hello World</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="img_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>读取灰度图</string>
    </property>
   </widget>
   <widget class="QLabel" name="opencv_img">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyFirstQt_VS.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>MyFirstQt_VSClass</receiver>
   <slot>exit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_hello_button</sender>
   <signal>clicked()</signal>
   <receiver>MyFirstQt_VSClass</receiver>
   <slot>button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>img_button</sender>
   <signal>clicked()</signal>
   <receiver>MyFirstQt_VSClass</receiver>
   <slot>img_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button_clicked()</slot>
  <slot>exit_clicked()</slot>
  <slot>img_clicked()</slot>
 </slots>
</ui>
